src/components/GroupSystems/GroupSystems.js

```diff
@@ -175,6 +180,7 @@ const GroupSystems = ({ groupName, groupId }) => {
                 requiredPermissions={REQUIRED_PERMISSIONS_TO_MODIFY_GROUP(
                   groupId,
                 )}
+                isAriaDisabled={isKesselEnabled ? ungrouped : false}
```

Comment:
This is effectively the same logic as what you used above, right?
`isAriaDisabled={isKesselEnabled && ungrouped}`

Reply:
true, I'll fix this in a minute